src/test/java/core/basesyntax/model/FruitRecordTest.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.model;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class FruitRecordTest {
+    private Fruit testFruit = new Fruit("test");
+    private FruitRecord fruitRecordExpected = new FruitRecord("b", testFruit, 10);
+
+    @Test
+    public void getOperation_validEntry_Ok() {
+        String expectedOperation = fruitRecordExpected.getOperation();
```

Comment:
Let's not use methods, that we test, to get expected result. You can just write the result you expect to see(fix it an all tests)